usersession/agent/rest_api_test.go

```diff
@@ -74,3 +95,265 @@ func (s *restSuite) TestSessionInfo(c *C) {
 		"version": "42b1",
 	})
 }
+
+func (s *restSuite) TestServiceControl(c *C) {
+	// the agent.Services end point only supports POST requests
+	c.Assert(agent.ServiceControlCmd.GET, IsNil)
+	c.Check(agent.ServiceControlCmd.PUT, IsNil)
+	c.Check(agent.ServiceControlCmd.POST, NotNil)
+	c.Check(agent.ServiceControlCmd.DELETE, IsNil)
```

Comment:
Maybe also add HEAD and POSTFORM?

Reply:
There are no HEAD or POSTFORM members on the command struct.